lib/index.ts

```diff
@@ -1,11 +1,15 @@
 import moment from 'moment'
-import { getLocale, translate } from '@nextcloud/l10n'
+import Gettext from 'node-gettext'
+import { getLocale } from '@nextcloud/l10n'
 
-function t(text: string): string {
-    return translate('nextcloud-moment', text)
-}
+const gt = new Gettext()
 
-moment.locale(getLocale())
+const locale = getLocale()
+LOCALES.map(data => {
+    gt.addTranslations(data.locale, 'messages', data.json)
+})
+gt.setLocale(locale)
```

Comment:
Shouldn't we use the language settings for this instead of the locale setting?

Reply:
I'm never sure. @nickvergessen knows best

Reply:
Well see https://github.com/nextcloud/server/pull/14784

So moment JS is actually "fucked up" as it has only one option to set this and is unable to handle a locale being different from the language.

Reply:
OK, then it should be fine for now otherwise it would just be the "seconds" which would be in the proper language, the rest provided by moment would have the language defined by the locale.